03week/towersOfHanoi.js

```diff
@@ -19,24 +19,52 @@ function printStacks() {
   console.log("c: " + stacks.c);
 }
 
-function movePiece() {
-  // Your code here
+function movePiece(startStack, endStack) {
+  //created two variable to. One is the array you are starting with and the other is the array you are ending in
+  let currentArray = stacks[startStack];
+  let toArray = stacks[endStack];
 
+  //used push to push whatever was popped from the currentArray onto the toArray (destination array)
+  toArray.push(currentArray.pop());
 }
 
-function isLegal() {
-  // Your code here
-
+function isLegal(startStack, endStack) {
+  //created currentArra and toArray variables for isLegal function
```

Comment:
It would have been good to check that the start array is not empty at the beginning of the function.